modules/items/client/views/admin/list-items.client.view.html

```diff
@@ -28,3 +25,7 @@ <h4 class="list-group-item-heading" data-ng-bind="item.title"></h4>
     No items yet, why don't you <a data-ui-sref="admin.items.create">create one</a>?
   </div>
 </section>
+
+<script>
```

Comment:
This seems very dirty. Why do we need this??

Reply:
My bad, I rush through the html sometimes.